### ISSUE TITLE
Locked cfg-if version conflicts with log 0.4.8 on mio 0.6.20

### ISSUE THREAD

**Issue by AbsurdlySuspicious** [tokio-rs/mio#1169]
```
$ cargo fetch

error: failed to select a version for `cfg-if`.
    ... required by package `mio v0.6.20`
    ... which is depended on by `tokio v0.2.0`
    ... which is depended on by (my project)
versions that meet the requirements `= 0.1.9` are: 0.1.9

all possible versions conflict with previously selected packages.

  previously selected package `cfg-if v0.1.10`
    ... which is depended on by `log v0.4.8`
    ... which is depended on by `fern v0.5.9`
    ... which is depended on by (my project)

failed to select a version for `cfg-if` which could resolve this conflict
```

**Comment by est31** [comment]
See #1170 for a fix.

**Comment by Thomasdezeeuw** [comment]
Fixed by #1170.
